Parsers/ASimUserManagement/ASimUserManagementLinuxAuthpriv.yaml

```diff
@@ -0,0 +1,342 @@
+  ),(
+      Syslog
+      | where not(disabled)
+      | where Facility == "authpriv"
+      | where ProcessName == "groupadd"
+          and SyslogMessage startswith "group added to "
+          and not(SyslogMessage has "GID=")
+      | parse SyslogMessage with "group added to " * "name=" GroupName
+      | extend 
+          EventType   = "GroupCreated",
+          EventResult = "Success"
+      | invoke ItemParser()
+  ),(
+      Syslog
+      | where not(disabled)
+      | where Facility == "authpriv"
+      | where ProcessName == "groupadd"
+          and SyslogMessage startswith "new group: name="
+      | parse SyslogMessage with "new group: name=" GroupName ", GID=" GroupId
+      | extend 
+          EventType   = "GroupCreated",
+          EventResult = "Success"
+      | invoke ItemParser()
+  ),(
+      Syslog
+      | where not(disabled)
+      | where Facility == "authpriv"
+      | where ProcessName == "groupadd"
+          and SyslogMessage startswith "cannot open login definitions"
+      | extend 
+          EventType          = "GroupCreated", 
+          EventResult        = "Failure",
+          EventResultDetails = "NotAuthorized"
+      | invoke ItemParser()
+  ),(
+      Syslog
+      | where not(disabled)
+      | where Facility == "authpriv"
+      | where ProcessName == "groupmod"
+          and SyslogMessage startswith "group changed in "
+      | parse SyslogMessage with "group changed in " * " (group " Temp_GroupName ", new name: " *
+      | extend 
+          split(Temp_GroupName, "/")
+      | extend 
+          GroupName = tostring(Temp_GroupName[0]),
+          GroupId   = tostring(Temp_GroupName[1])
+      | project-away Temp_GroupName
+      | extend 
+          EventType   = "GroupModified",
+          EventResult = "Success"
+      | invoke ItemParser()
+  ),(
+      Syslog
+      | where not(disabled)
+      | where Facility == "authpriv"
+      | where ProcessName == "groupmod"
+          and SyslogMessage startswith "failed to change "
+      | parse SyslogMessage with "failed to change " * " (group " Temp_GroupName ", new name: " *
+      | extend split(Temp_GroupName, "/")
+      | extend 
+          GroupName = tostring(Temp_GroupName[0]),
+          GroupId   = tostring(Temp_GroupName[1])
+      | project-away Temp_GroupName
+      | extend 
+          EventType   = "GroupModified",
+          EventResult = "Failure"
+      | invoke ItemParser()
+  ),(
+      Syslog
+      | where not(disabled)
+      | where Facility == "authpriv"
+      | where ProcessName == "groupdel"
+      | parse SyslogMessage with "group '" GroupName "' removed" *
+      | extend 
+          EventType   = "GroupDeleted",
+          EventResult = "Success"
+      | invoke ItemParser()
+  ),(
+      Syslog
+      | where not(disabled)
+      | where Facility == "authpriv"
+      | where ProcessName == "gpasswd"
+      | parse SyslogMessage with "user " TargetUsername " " Action " by " ActorUsername " " * " group " GroupName
```

Comment:
most ActorUsername are coming as null. Can we say ActorUsername = TargetUsername where its applicable?
![image](https://github.com/Azure/Azure-Sentinel/assets/97222872/da96bfb3-4a5c-49ca-9ae6-8b57a746a576)

Reply:
ActorUsername is mandatory field

Reply:
We can't do that because TargetUsername is the name the account where the action is performed on, ie. useradd or userdelete. I think ActorUsername should be recommended.